src/supported_apps.ts

```diff
@@ -277,4 +277,16 @@ export const supportedApps: SubstrateAppParams[] = [
     slip0044: 0x80000219,
     ss58_addr_type: 126,
   },
+  {
+    name: "Enjin",
+    cla: 0xb9,
+    slip0044: 0x80000483,
+    ss58_addr_type: 2135,
+  },
+  {
+    name: "Matrixchain",
```

Comment:
[Here](https://github.com/paritytech/ss58-registry/blob/main/ss58-registry.json) the chain seems to be called `Efinity`. Why `Matrixchain?

Reply:
It was changed https://enjin.io/blog/welcome-enjin-blockchain#:~:text=From%20Efinity%20to%20Enjin,ENJ%2C%20ensuring%20a%20smooth%20transition.
But did not submit to those PR yet, will do that now then, thanks

Reply:
Here it is: https://github.com/paritytech/ss58-registry/pull/187